api/src/pcapi/routes/backoffice_v3/offers/forms.py

```diff
@@ -0,0 +1,256 @@
+import enum
+import json
+import typing
+
+from flask_wtf import FlaskForm
+import wtforms
+
+from pcapi.core.categories import categories
+from pcapi.models.offer_mixin import OfferValidationStatus
+from pcapi.routes.backoffice_v3 import autocomplete
+from pcapi.routes.backoffice_v3 import filters
+from pcapi.routes.backoffice_v3.forms import constants
+from pcapi.routes.backoffice_v3.forms import empty as empty_forms
+from pcapi.routes.backoffice_v3.forms import fields
+from pcapi.routes.backoffice_v3.forms import utils
+
+
+class SearchOperators(enum.Enum):
+    EQUALS = "est exactement"
+    NOT_EQUALS = "n'est pas exactement"
```

Comment:
```suggestion
    NOT_EQUALS = "n'est pas"
```
Ou sinon, `est égal à` et `est différent de`